### ISSUE TITLE
memo doesn't work when there's no param in member expression call expression

### ISSUE THREAD

**Issue by mdynnl** [LXSMNSYC/solid-labels#7]
sorry for weird description
the code example should be self-explanatory

```javascript

let count = $signal(0)
const double = $memo(count * 2) // works
const hex = $memo(double.toString(16)) // does works
$: var str = hex.toString()
// gets compiled to
// const str = createMemo (hex().toString)
```
CTF, labels, comments all the same

**Comment by LXSMNSYC** [comment]
Good catch, I'll check if I had some optimization step that might be including this unintentionally

**Comment by LXSMNSYC** [comment]
This should be fixed in `0.11.1`
